backend/backend.py

```diff
@@ -76,7 +87,10 @@ def on_loop(self) -> None:
         with self.exception_handler:
             self._on_loop()
 
-        if time() > self.prev_recording_status_time + self.recording_status_interval:
+        if (
+            time()
+            > self.prev_recording_status_time + self.recording_status_interval
```

Comment:
Not sure how I feel about this being separated... lmao

Reply:
It just means this line should be refactored to:

```python
update_time = self.prev_recording_status_time + self.recording_status_interval
if time() > update_time: 
	...
```